tests/e2e.spec.ts

```diff
@@ -6,14 +6,22 @@ import { decodeTxData } from "../src/decode";
 
 dotenv.config();
 
+const TESTNET_ROOT_KEY =
+  "secp256k1:4NfTiv3UsGahebgTaHyD9vF8KYKMBnfd6kh94mK6xv8fGBiJB8TBtFMP5WWXz6B89Ac1fbpzPwAvoyQebemHFwx3";
+const MAINNET_ROOT_KEY =
+  "secp256k1:3tFRbMqmoa6AAALMrEFAYCEoHcqKxeW38YptwowBVBtXK1vo36HDbUWuR6EZmoK4JcH6HDkNMGGqP1ouV7VZUWya";
+
 describe("Near Safe Requests", () => {
   let adapter: NearSafe;
   beforeAll(async () => {
+    const networkId = "testnet";
```

Comment:
This is a bit silly, but I wanted to explicitly show there are two possible keys.